doc/nrf/software_maturity.rst

```diff
@@ -128,24 +128,28 @@ The following table indicates the software maturity levels of the support for th
       * - **Unicast source**
         - BAP unicast, one Connected Isochronous Group (CIG) with two Connected Isochronous Streams (CIS).
 
-          Transmitting unidirectional audio using CIG and CIS.
+          Transmitting unidirectional or transceiving bidirectional audio using CIG and CIS.
 
           Play and pause emulated by disabling and enabling stream, respectively.
         - The following limitations apply:
 
           * BAP unicast, one CIG with two CIS.
           * Audio input: USB or I2S (Line in or using Pulse Density Modulation).
+          * Audio output: USB or I2S/Analog headset output.
```

Comment:
Isnt it a limitation that only one headset can be used for bidirectional stream?